create.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App Create</title>
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body>
+    <h1>Create</h1>
+    <main>
+      <form aria-labelledby="create-question-heading" autocoplete="off">
+        <label for="input-question">Let us know your question:</label>
+        <textarea
+          id="input-question"
+          cols="20"
+          rows="2"
+          required
+          placeholder="What is your question?"
+        >
+        </textarea>
+        <label for="input-answer">Let us know your answer:</label>
+        <textarea
+          id="input-answer"
+          cols="40"
+          rows="10"
+          required
+          placeholder="What is the answer?"
+        >
+        </textarea>
+        <label for="input-tags">Please name tags:</label>
+        <textarea
+          id="input-tags"
+          cols="40"
+          rows="1"
+          required
+          placeholder="Tag 1, Tag 2, Tag 3, Tag 4"
+        >
+        </textarea>
```

Comment:
here i would recommend more an `input` instead of an `textarea`

Reply:
Oh thank you Michi, I think you are absolutely right 😊